persistence/assetlist.json

```diff
@@ -645,6 +645,47 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/huahua.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/huahua.svg"
       }
+    },
+    {
+      "description": "PSTAKE Liquid-Staked XPRT",
+      "denom_units": [
+        {
+          "denom": "stk/uxprt",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "stkxprt",
+          "exponent": 6,
+          "aliases": [
+            "stk/xprt"
+          ]
+        }
+      ],
+      "base": "stk/uxprt",
+      "name": "PSTAKE staked XPRT",
+      "display": "stkxprt",
+      "symbol": "stkXPRT",
+      "traces": [
+        {
+          "type": "liquid-stake",
+          "counterparty": {
+            "chain_name": "persistence",
+            "base_denom": "uxprt"
+          },
+          "provider": "pSTAKE"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/persistence/images/stkxprt.png",
```

Comment:
Could I ask for a better png file? It looks blurry, has a shadow, and has extra padding. In contrast, the svg looks great. We aim to have the images within the same object have to exact same appearance.

Reply:
Sure. Updated it. Can you please check if the updated image works? @JeremyParish69